public/app/features/search/components/DashboardListPage.tsx

```diff
@@ -46,22 +45,16 @@ export const DashboardListPage: FC<Props> = memo(({ navModel, match, location })
 
   return (
     <Page navModel={value?.pageNavModel ?? navModel}>
-      {Boolean(config.featureToggles.panelTitleSearch) ? (
-        <Page.Contents
-          isLoading={loading}
-          className={css`
-            display: flex;
-            flex-direction: column;
-            overflow: hidden;
-          `}
-        >
-          <ManageDashboardsNew folder={value?.folder} />
-        </Page.Contents>
-      ) : (
-        <Page.Contents isLoading={loading}>
-          <ManageDashboards folder={value?.folder} />
-        </Page.Contents>
-      )}
+      <Page.Contents
+        isLoading={loading}
+        className={css`
+          display: flex;
+          flex-direction: column;
+          overflow: hidden;
+        `}
+      >
+        <ManageDashboardsNew folder={value?.folder} />
```

Comment:
keeping this as `...New` for now, just to make git not show the diff between old vs new